openlibrary/core/bookshelves.py

```diff
@@ -121,7 +121,7 @@ def get_users_logged_books(cls, username, bookshelf_id, limit=100, page=1):
 
         username (str) - who logged this book
         work_id (int) - the Open Library work ID as an int (e.g. OL123W becomes 123)
-        bookshelf_id (int) - the ID of the bookshelf, see: PRESET_BOOKSHELVES
+        bookshelf_id (int) - the ID of the bookshelf, see: PRESET_BOOKSHELVES. If bookshelf_id = 0 then return books from all bookshelves.
```

Comment:
Readability: Lines of Python code should be 88 characters max.
```suggestion
        bookshelf_id (int) - the ID of the bookshelf, see: PRESET_BOOKSHELVES.
            If bookshelf_id = 0 then return books from all bookshelves.
```